pkg/controller/nodeipam/ipam/cidr_allocator.go

```diff
@@ -116,6 +128,15 @@ func New(kubeClient clientset.Interface, cloud cloudprovider.Interface, nodeInfo
 	switch allocatorType {
 	case RangeAllocatorType:
 		return NewCIDRRangeAllocator(kubeClient, nodeInformer, allocatorParams, nodeList)
+	case MultiCIDRRangeAllocatorType:
+		if !utilfeature.DefaultFeatureGate.Enabled(features.MultiCIDRRangeAllocator) {
+			return nil, fmt.Errorf("invalid CIDR allocator type: %v, feature gate %v must be enabled", allocatorType, features.MultiCIDRRangeAllocator)
+		}
+		if err != nil {
+			return nil, err
+		}
```

Comment:
this is a leftover, we don't need to check here again

Reply:
Oops, removed!